bigtable/admin.go

```diff
@@ -217,6 +217,9 @@ type TableConf struct {
 	SplitKeys []string
 	// Families is a map from family name to GCPolicy
 	Families map[string]GCPolicy
+	// Set to true to make the table protected against data loss
+	// i.e. deleting the table, the column families in the table and the instance containing the table would be prohibited
+	DeletionProtection *bool
```

Comment:
Why is this a pointer to a bool rather than just a bool?

Reply:
I believe this field is optional. Again, I think we should use a different struct update.